code/modules/projectiles/projectile.dm

```diff
@@ -1212,20 +1212,13 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 		//Note, in the case of limbs missing, this will increase average armor if remaining armor is higher than if fully limbed.
 		return armor_val / total_weight
 
-/mob/living/carbon/xenomorph/get_soft_armor(armor_type, proj_def_zone)
-	return ..() * get_sunder()
```

Comment:
This might also lead to some issues with things that grab xeno armor manually instead of going through modify_by_armor. I'm pretty sure there were a few of those, though I do not recall where.